src/checks.rs

```diff
@@ -1705,6 +1715,7 @@ impl CheckKind {
                 | CheckKind::UnnecessaryGeneratorDict
                 | CheckKind::UnnecessaryGeneratorList
                 | CheckKind::UnnecessaryGeneratorSet
+                | CheckKind::UnnecessaryLRUCacheParams
```

Comment:
@chammika-become - I added `CheckKind::UnnecessaryLRUCacheParams` to this `fixable` list to ensure that it's marked as such in the `README.md`.